google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITObjectTest.java

```diff
@@ -1069,28 +1008,6 @@ public void testCopyBlobFail() {
     }
   }
 
-  @Test
-  public void testReadAndWriteChannels() throws IOException {
```

Comment:
I feel like this is covered via the various other Read/Write channel tests

Reply:
Agreed, this isn't doing anything unique.